rdmo/questions/viewsets.py

```diff
@@ -117,6 +118,11 @@ def get_export_renderer_context(self, request):
         }
 
 
+class CatalogToggleCurrentSiteViewSet(ElementToggleCurrentSiteViewSetMixin):
```

Comment:
Why is this a new ViewSet and not part of the big CatalogViewSet, the route would look nicer `/api/v1/questions/catalog/1/add-site/`.

Reply:
yes, I think I started with trying to define it as a ViewSet action but could not get the permissions to apply.. 
agreed, I'll look and try again.

Reply:
it works as mixin for the actions, thanks!